### DISCUSSION TITLE
Replacing gnome-calc with rpncalc

### DISCUSSION THREAD

**Discussion by benjaminbellamy** [frossm/rpncalc#2] [Show and tell]
I never use gnome-calc so I find it convenient to replace it with rpncalc (so that all gnome-calc shortcuts launch rpncalc):
```
$ sudo mv /usr/bin/gnome-calculator /usr/bin/gnome-calculator.bak
```
Then `$ sudo vim /usr/bin/gnome-calculator`:
```
#!/bin/bash

gnome-terminal -x /snap/bin/rpncalc
```
And `$ sudo chmod +x /usr/bin/gnome-calculator`

Et voilà !

**Comment by frossm** [comment]
Great idea Benjaminbellamy!  And thanks for using RPNCalc.
